contracts/contracts/core/PreconfManagerStorage.sol

```diff
@@ -13,9 +13,6 @@ abstract contract PreconfManagerStorage {
     /// @dev Address of the oracle contract
     address public oracleContract;
 
-    /// @dev The number of blocks per window
-    uint256 public blocksPerWindow;
```

Comment:
don't delete for upgrading

Reply:
We can delete or add anything to storage contracts in main. Release branch is where upgrade safety should be considered